### ISSUE TITLE
ign-tools fails to configure on mac os big sur arm64 (m1)

### ISSUE THREAD

**Issue by YVbakker** [ignitionrobotics/ign-tools#51]
## Environment
* OS Version: Mac OS big sur
* Source build: tags/1.2.0
    
<details>
  
```
Compiling the C compiler identification source file "CMakeCCompilerId.c" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc 
Build flags: 
Id flags:  

The output was:
1
ld: library not found for -lSystem
clang: error: linker command failed with exit code 1 (use -v to see invocation)


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ 
Build flags: 
Id flags:  

The output was:
1
ld: library not found for -lc++
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

</details>

## Description
* Expected behavior: configure successfully 
* Actual behavior: fails to configure

## Steps to reproduce

1. Download source tarball from github && tar xf
2. mkdir build && cd build
3. cmake .. -DCMAKE_INSTALL_PREFIX=/usr

## Output

See above and full trace:
https://gist.github.com/YVbakker/32d686da43288c5817e644caaa1f5ea6

**Comment by YVbakker** [comment]
I think I found the root of the problem in findSSE.cmake. Adding an extra check whether we're on x86 or not fixes this error. Can someone else verify it builds successfully without?

**Comment by j-rivero** [comment]
Hopefully fixed by #55
